pkg/serverless/executioncontext/executioncontext.go

```diff
@@ -218,6 +218,7 @@ func (ec *ExecutionContext) RestoreCurrentStateFromFile() error {
 	ec.lastLogRequestID = restoredExecutionContextState.LastLogRequestID
 	ec.lastOOMRequestID = restoredExecutionContextState.LastOOMRequestID
 	ec.coldstartRequestID = restoredExecutionContextState.ColdstartRequestID
+	ec.wasColdStart = restoredExecutionContextState.WasColdStart
```

Comment:
I believe there are tests for this method. Should update them to include the new field.